backends/cuda/ceed-cuda-qfunction.c

```diff
@@ -177,3 +177,10 @@ int CeedQFunctionCreate_Cuda(CeedQFunction qf) {
                                 CeedQFunctionDestroy_Cuda); CeedChk(ierr);
   return 0;
 }
+
+int CeedQFunctionCreateIdentity_Cuda(CeedQFunction qf) {
+  int ierr;
+  Ceed ceed;
+  ierr = CeedQFunctionGetCeed(qf, &ceed); CeedChk(ierr);
+  return CeedError(ceed, 1, "Backend does not implement Identity QFunctions");
```

Comment:
Surely it can be implemented in a naive way (write a normal QFunction).

Reply:
Copying over the vectors with `CEED_COPY_VALUES` is probably the easiest

Reply:
updated